source/tutorials/configure-selinux/configure-selinux.rst

```diff
@@ -0,0 +1,139 @@
+.. _configure-selinux:
+
+SELinux Configuration and Rules
+###############################
+
+Security-Enhanced Linux (`SELinux <https://source.android.com/security/selinux>`_) is enabled in Android to enforce the Mandatory Access Control for security. SELinux supports two working modes: permissive and enforcing:
+
+* In **permissive** mode, it only audits the operations of all domains and prints the AVC (Access Vector Cache) errors that violate the sepolicy rules, but it never blocks any operations.
+
+* In **enforcing** mode, it not only prints out the AVC errors, but also blocks the operations that violate the sepolicy rules.
```

Comment:
Suggested rewrite: 
In **enforcing** mode, it prints out the AVC errors and also blocks the operations that violate the sepolicy rules.